hack/dockerfiles/gosec.Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax=docker/dockerfile:1.3
+
+FROM securego/gosec AS validate
```

Comment:
Define it with a version that can be reset with `ARG` if needed.

Reply:
Done.